redash/handlers/metr_widgets.py

```diff
@@ -0,0 +1,34 @@
+
+from flask import request
+
+from redash import models
+from redash.handlers.base import BaseResource
+from redash.permissions import (
+    require_object_modify_permission,
+    require_permission
+)
+from redash.serializers import serialize_widget
+
+class metrWidgetTagsResource(BaseResource):
+    """
+    Resource for handling tags of a metrWidget.
+
+    This resource provides methods for retrieving and updating tags associated with a metrWidget.
+    """
+
+    @require_permission("edit_dashboard")
+    def post(self, widget_id):
+        widget = models.Widget.get_by_id_and_org(widget_id, self.current_org)
+        require_object_modify_permission(widget.dashboard, self.current_user)
+        widget_properties = request.get_json(force=True)
+        tags = widget_properties["tags"]
+
+        metrWidget=models.metrWidget.query.filter(models.metrWidget.widget==widget).first()
+        if metrWidget is None:
```

Comment:
Instead of doing the query manually, can't you use the back relation?

```suggestion
        if not widget.metrWidget: # This is assuming that you can use a 1to1 relation. Otherwise you have to keep the is None
```

Reply:
thanks, the update is also included[ in this commit ](https://github.com/metr-systems/redash/pull/4/commits/7f337577ff4cca5ee75b2645d8884daf85ab7abd)